app/models/manageiq/providers/container_manager.rb

```diff
@@ -19,6 +19,7 @@ class ContainerManager < BaseManager
     has_many :container_build_pods, :foreign_key => :ems_id, :dependent => :destroy
     has_many :container_templates, :foreign_key => :ems_id, :dependent => :destroy
     has_one :container_deployment, :foreign_key => :deployed_ems_id, :inverse_of => :deployed_ems
+    has_many :miq_alert_statuses, :through => :container_nodes
```

Comment:
I think we can handle that when we add more entities?

What I was thinking of doing when we add more container entities that have miq_alert_statuses is writing custom code for the join that will look at multiple tables (I think conceptually join is the best solution here).

@kbrock is there a better solution? we will need a way to has_many through multiple tables basically.
(=== ```has_many :miq_alert_statuses, :through => [:container_nodes, :container_groups ...]```)